docker/django/Dockerfile

```diff
@@ -76,6 +78,13 @@ RUN tar xvfz courtlistener.tar.gz && \
 # `python-base` sets up all our shared environment variables
 FROM build-${BUILD_ENV} as python-base
 
+# Collect compiled assets from webpack-build stage.
+# Note: Always produce production code and never produce source-maps
+# (run the devMiddleware for legible output)
+WORKDIR /opt/courtlistener/cl
+RUN npm install
+RUN npx webpack --mode=production --no-devtool
+
```

Comment:
Note: You can see what that would look like in the alternative branch, [javascript-webpack-staged](https://github.com/cweider/courtlistener/commit/8207c6744d8ce74f713f45593655eeacb840ae39)’s version of the [Dockerfile](https://github.com/cweider/courtlistener/commit/8207c6744d8ce74f713f45593655eeacb840ae39#diff-10aefcba0ca0bff173d169c2b0368a7b669159cd9d7b565a9d7ccc020919d41d). Unfortunately, this is not compatible with the repository unarchiving approach used in the `build-prod` step – which would need the change [replace use of repository archive during build-prod](https://github.com/freelawproject/courtlistener/commit/518b7ef542b5620dccbc2bdb09ee9a0e68d19a5a).